cmd/notary/cert.go

```diff
@@ -159,6 +166,51 @@ func (c *certCommander) certRemove(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// certRotate replaces a certificate with a new version
+func (c *certCommander) certRotate(cmd *cobra.Command, args []string) error {
```

Comment:
This appears to always rotate all root keys (or possibly all those for which we hold the private key?). Is that the behaviour we want?